website/docs/hail-format.md

```diff
@@ -15,6 +15,13 @@ These are also available on Amazon S3:
 - Summary statistics MatrixTable: `s3://pan-ukb-us-east-1/sumstats_release/results_full.mt` (12.78 T)
 - Meta-analysis MatrixTable: `s3://pan-ukb-us-east-1/sumstats_release/meta_analysis.mt` (12.54 T)
 
+In addition, in-sample full LD matrices and scores are available on Amazon S3:
+- LD BlockMatrix `s3://pan-ukb-us-east-1/ld_release/UKBB.{pop}.ldadj.bm` (43.3 T in total)
```

Comment:
Can you maybe write out how large each population's is?

Reply:
addressed (in the next line)